boards/wsn430-v1_4/include/board.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_wsn430-v1_3b WSN430 v1.3b
+ * @ingroup     boards
+ * @brief       Support for the Senslab WSN430 v1.3b board
+ *
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the Senslab WSN430 v1.3b board
+ *
+ * @author      Martine Lenders <m.lenders@fu-berlin.de>
+ */
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "board_common.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Definition of the interface to the CC1101 radio
+ */
+#define CC2420_PARAMS_BOARD     {   .spi        = SPI_DEV(0),       \
+                                    .spi_clk    = SPI_CLK_5MHZ,     \
+                                    .pin_cs     = GPIO_PIN(P4, 2),  \
+                                    .pin_fifo   = GPIO_PIN(P1, 3),  \
+                                    .pin_fifop  = GPIO_PIN(P1, 4),  \
+                                    .pin_cca    = GPIO_PIN(P1, 6),  \
+                                    .pin_sfd    = GPIO_PIN(P1, 5),  \
+                                    .pin_vrefen = GPIO_PIN(P3, 0),  \
+                                    .pin_reset  = GPIO_PIN(P1, 7)   }
```

Comment:
I've taken those from the [WSN430 1.4 schematics](https://github.com/iot-lab/iot-lab/wiki/Hardware_Wsn430-node#schematics-and-datasheets).